temoa/extensions/get_comm_tech.py

```
@@ -28,7 +29,9 @@ def get_tperiods(inp_f):
     for row in cur:
         x.append(row[0])
     for y in x:
-        cur.execute("SELECT DISTINCT period FROM output_flow_out WHERE scenario is '" + str(y) + "'")
+        cur.execute(
+            "SELECT DISTINCT period FROM output_flow_out WHERE scenario is '" + str(y) + "'"
```

Ruff (0.14.5)

Possible SQL injection vector through string-based query construction

(S608)